test/base.js

```diff
@@ -14,6 +14,8 @@ var Promise = require('pinkie-promise');
 var yeoman = require('yeoman-environment');
 var userHome = require('user-home');
 
+var LF = require('os').EOL;
```

Comment:
Let's drop the LF variable.

Using the Node core constant make the code predictable. By comparison, if I read `LF`, I'll wonder what this value is and where it comes from.